File: Objective_C/Projects/Phase1_Exam/Phase1_Exam/ViewController.m
//
//  ViewController.m
//  Phase1_Exam
//
//  Created by kongyunpeng on 12/27/15.
//  Copyright © 2015 kongyunpeng. All rights reserved.
//

#import "ViewController.h"
#import "NextViewController.h"
@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *showLbl;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    NextViewController *controller = segue.destinationViewController;
    controller.setShowLbl = ^(NSString *text) {
        _showLbl.text = text;
    };
    
}
@end

File: Swfit/Projects/Initialization/Initialization/ViewController.swift
//
//  ViewController.swift
//  Initialization
//
//  Created by kongyunpeng on 12/26/15.
//  Copyright © 2015 kongyunpeng. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
       let male = Male(handsome: true)
       male.printMale()
        
        let uglyMan = Male(uglyMan: false)
        uglyMan.printMale()
        
    let vehicle = Vehicle()
    print(vehicle.description)
    
    let bicycle = Bicycle()
        print(bicycle.description)
    let bicycle2 = Bicycle(OneWheel: 1)
        print(bicycle2.description)
    }
  
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

